wagtail_airtable/templates/wagtailsnippets/snippets/index.html

```diff
@@ -2,40 +2,45 @@
-    {% fragment as action_text_snippet %}{% blocktrans trimmed with snippet_type_name=model_opts.verbose_name %}Add {{ snippet_type_name }}{% endblocktrans %}{% endfragment %}
+    {% wagtail_major_version as wagtail_major_version %}
+    {% if wagtail_major_version < 6 %}
+        {% include 'wagtailadmin/shared/headers/slim_header.html' %}
+        {% fragment as base_action_locale %}{% if locale %}{% include 'wagtailadmin/shared/locale_selector.html' with theme="large" %}{% endif %}{% endfragment %}
+        {% fragment as action_url_add_snippet %}{% if can_add_snippet %}{% url view.add_url_name %}{% if locale %}?locale={{ locale.language_code }}{% endif %}{% endif %}{% endfragment %}
+        {% fragment as action_text_snippet %}{% blocktrans trimmed with snippet_type_name=model_opts.verbose_name %}Add {{ snippet_type_name }}{% endblocktrans %}{% endfragment %}
 
-    {% fragment as extra_actions %}
-        {% if view.list_export %}
-            {% include view.export_buttons_template_name %}
-        {% endif %}
-	{# begin wagtail-airtable customisation for Wagtail 5.2 #}
-        {% can_import_model model_opts.label as can_import_model %}
-        {% if can_add_snippet and can_import_model %}
-            <form action="{% url 'airtable_import_listing' %}" method="post">
-                {% csrf_token %}
-                <input type="hidden" name="model" value="{{ model_opts.label }}" />
-                <input type="hidden" name="next" value="{{ request.path }}" />
-                <button type="submit" class="button button-longrunning" data-controller="w-progress" data-action="w-progress#activate" data-w-progress-active-value="{% trans "Importing..." %}">
-		    {% icon name="spinner" %}
-                    <em data-w-progress-target="label">{% blocktrans with snippet_type_name=model_opts.verbose_name_plural %}Import {{ snippet_type_name }}{% endblocktrans %}</em>
-                </button>
-            </form>
-        {% endif %}
-	{# end wagtail-airtable customisation #}
-    {% endfragment %}
+        {% fragment as extra_actions %}
+            {% if view.list_export %}
+                {% include view.export_buttons_template_name %}
+            {% endif %}
+        {# begin wagtail-airtable customisation for Wagtail 5.2 #}
+            {% can_import_model model_opts.label as can_import_model %}
+            {% if can_add_snippet and can_import_model %}
+                <form action="{% url 'airtable_import_listing' %}" method="post">
+                    {% csrf_token %}
+                    <input type="hidden" name="model" value="{{ model_opts.label }}" />
+                    <input type="hidden" name="next" value="{{ request.path }}" />
+                    <button type="submit" class="button button-longrunning" data-controller="w-progress" data-action="w-progress#activate" data-w-progress-active-value="{% trans "Importing..." %}">
+                {% icon name="spinner" %}
```

Comment:
nitpick: underindented.
```suggestion
                        {% icon name="spinner" %}
```